python/observability-online-eval/openai/computer-use/agent.py

```
@@ -0,0 +1,157 @@
+from typing import List, Literal
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)